Reqnroll.VisualStudio.ProjectTemplate/ProjectTemplate.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Reqnroll.NUnit" Version="2.0.2" />
     <PackageReference Include="nunit" Version="3.14.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />$endif$$if$ ('$unittestframework$' == 'MSTest')
-    <PackageReference Include="Reqnroll.MsTest" Version="1.0.0" />
+    <PackageReference Include="Reqnroll.MsTest" Version="2.*" />
```

Comment:
This would make an easy fix to avoid always updating the template, but this would generate project files with the `*` style dependencies, that are pretty unsafe (and it also might confuse the VS extension, because I could not decide which Reqnroll version it is - causing problems and obscure our telemetry). 

So I would not do that. 

Ideally the VS extension should detect somehow the latest Reqnroll version and inject that explicitly to the project file. But for now hard-coding our latest would be still useful.

Reply:
OK. I've updated the PR to explicitly refer to version 2.0.2 just like it does for the other test frameworks.